spvpatch.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/spirv_msl.cpp b/spirv_msl.cpp
```

Comment:
?

Reply:
Whoops, that's another change I've been toying with but don't have the full understanding yet to commit. Shouldn't have been included, not sure how it got caught up, will update pull request shortly to remove it.